### PR TITLE
AMP-28044: Use query instead of path parameter

### PR DESCRIPTION
**Pull request by developster**

### CODE REVIEW

File: amp/TEMPLATE/ampTemplate/gisModule/dev/app/js/amp/data/collections/activity-collection.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ module.exports = Backbone.Collection
 
     if (!_.isEmpty(aryOfIDs)) {
       // do an api request to get remaining ones
-      this.url = '/rest/gis/activities/' + aryOfIDs.join(',');
+      this.url = '/rest/gis/activities' + aryOfIDs.map(function(id) { return "id="+id; }).join('&');
```
Comment (jdeanquin-dg) [inline amp/TEMPLATE/ampTemplate/gisModule/dev/app/js/amp/data/collections/activity-collection.js]: you have a missing ? in the URL

File: amp/TEMPLATE/ampTemplate/gisModule/dev/app/js/amp/data/collections/activity-collection.js [hunk 2]
```diff
@@ -243,7 +243,7 @@ module.exports = Backbone.Collection
 			var activities = new TempActivityCollection([], {
 				appData : this.appData
 			});
-			activities.url = '/rest/gis/activities/' + aryOfIDs.join(',');
+			activities.url = '/rest/gis/activities?' + aryOfIDs.map(function(id) { return "id="+id; }).join('&');
```
Comment (jdeanquin-dg) [inline amp/TEMPLATE/ampTemplate/gisModule/dev/app/js/amp/data/collections/activity-collection.js]: here too there is a missing ?

Comment (developster) [reply amp/TEMPLATE/ampTemplate/gisModule/dev/app/js/amp/data/collections/activity-collection.js]: Actually here question mark is present. When I tested apparently the other code wasn't triggered.

File: amp/TEMPLATE/ampTemplate/gisModule/dev/app/js/amp/data/collections/activity-collection.js (hunk 1 above)
Comment (developster) [reply amp/TEMPLATE/ampTemplate/gisModule/dev/app/js/amp/data/collections/activity-collection.js]: Good catch!

File: amp/TEMPLATE/ampTemplate/gisModule/dev/app/js/amp/data/collections/activity-collection.js (hunk 2 above)
Comment (jdeanquin-dg) [reply amp/TEMPLATE/ampTemplate/gisModule/dev/app/js/amp/data/collections/activity-collection.js]: yeah you are right, I must have been looking the previous one

**Pull request merged**